assets/css/form.css

```diff
@@ -0,0 +1,123 @@
+/* css variables - contact form section*/
+
+.Sec-3 {
+    background: #F4F5F7;
+    display: flex;
+    flex-flow: row wrap;
+    width: 100%;
+    height: 435px;
+    flex-shrink: 0;
+}
+
+
+
+.container { /* Image */
+    position: relative;
+    text-align: center;
+    width: 90%
+    height: 25%;
+    flex-shrink: 0;
+    transform: translate(-22%);
+    bottom: 13%;
+    
+}
+
+.centered { /* Unlock */
+    position: absolute;
+    top: 42%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    font-size: 350%;
+    color: #FFF;
+    font-family: Public Sans;
+    font-style: normal;
+    font-weight: 300;
+    line-height: 64px;
+    letter-spacing: -1px;
+}
+
+.Request { 
+    position: absolute;
+    color: #9597A5;
+    font-family: Public Sans;
+    font-size: 2rem;
+    font-style: normal;
+    font-weight: 300;
+    line-height: 38px;
+    letter-spacing: -1px;
+    width: 35%;
+    text-align: left;
+    align-items: center;
+    transform: translate(150%);
+    bottom: -100%;
+    
+}
+
+.Contact-form {
+    position: relative;
+    background: #fff;
```

Comment:
very minor but maybe you could have added a background colour to the input boxes so that it matches the figma design